src/Host.cpp

```diff
@@ -475,10 +477,9 @@ void Host::loadMap()
     }
 }
 
-void Host::startMapAutosave()
+void Host::startMapAutosave(int interval)
 {
-    auto settings = mudlet::self()->getQSettings();
-    if (auto interval = settings->value("autosaveIntervalMinutes", 2).toInt(); interval > 0) {
+    if (interval > 0) {
```

Comment:
Should there not be code to handle the `<1` case to run `killTimer(id)` on the (greater than 0) value that the `(int) QObject::startTimer(int interval, Qt:TimerType = Qt::CoarseTimer)` call returns?

Reply:
Not really, this function is only ever called once on Mudlet start

Reply:
Yeah, but if we ever get into a situation where we want to debug this during a run there is no way to do that as we are not capturing the returned value - so we cannot stop it with the current implementation in any way after it has been started...

Reply:
I don't think we need to build extensive debugging facilities for this, this code did not need debugging often - and when it will need it, well it's a 2min job to build it. Also this would definitely be scope creep as it's not solving any problem related to spellchecking 😅